config/routes.rb

```diff
@@ -8,6 +8,7 @@
   end
   get "static_pages/home"
 
+  resources :actor_categories, except: [:edit, :update]
   resources :actor_measures, only: [:index, :show]
```

Comment:
@parndt unrelated, but also I am wondering about the role of `only` here for `actor_measures` (as well as on `measure_actors`) as these should also allow creating and deleting via the API. Also I notice that the `create` and `destroy` operations are also missing in the controllers: 
- https://github.com/dumparkltd/marine-defrag-server/blob/master/app/controllers/measure_actors_controller.rb
- https://github.com/dumparkltd/marine-defrag-server/blob/master/app/controllers/actor_measures_controller.rb

Or maybe I am missing something?

Reply:
resolving...

see #20, #21